prefactor.cwl

```diff
@@ -15,62 +15,86 @@ inputs:
 outputs:
   losoto_h5:
     type: File
+    format: h5parm
+    label: "H5Parm solutions"
     outputSource: h5imp_cal/losoto_h5
 
   dTEC_1st:
     type: File
+    format: numpy
+    label: "TEC solutions"
     outputSource: fitclock/dTEC_1st
 
   dTEC_1st_sm:
     type: File
+    format: numpy
+    label: "Smoothed TEC solutions"
     outputSource: fitclock/dTEC_1st_sm
 
   dclock_1st:
     type: File
+    format: numpy
+    label: "Clock solutions"
     outputSource: fitclock/dclock_1st
 
   dclock_1st_sm:
     type: File
+    format: numpy
+    label: "Smoothed clock solutions"
     outputSource: fitclock/dclock_1st_sm
 
   amplitude_array:
     type: File
+    format: numpy
+    label: "Amplitude solutions"
     outputSource: ampl/amplitude_array
 
   dtec_allsols:
     type: File
+    format: image/png
```

Comment:
Should be `iana:image.png` with `iana: https://www.iana.org/assignments/media-types/` added under `$namespaces`